pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -1629,14 +1629,20 @@ void Object_beginTypeContext(JSOBJ _obj, JSONTypeContext *tc) {
     if (PyLong_Check(obj)) {
         PRINTMARK();
         tc->type = JT_LONG;
-        GET_TC(tc)->longValue = PyLong_AsLongLong(obj);
-
+        int _overflow = 0;
```

Comment:
```suggestion
        int overflow = 0;
```

No need to prepend an underscore here; may be confusing to authors who expect that to mean something